js/src/Ice/sources.json

```diff
@@ -6,14 +6,11 @@
         "Ice/BuiltinSequences.ice",
         "Ice/Identity.ice",
         "Ice/Locator.ice",
-        "Ice/LocatorF.ice",
         "Ice/Metrics.ice",
         "Ice/Process.ice",
-        "Ice/ProcessF.ice",
         "Ice/PropertiesAdmin.ice",
         "Ice/RemoteLogger.ice",
         "Ice/Router.ice",
-        "Ice/RouterF.ice"
```

Comment:
The last element of this list can't end with a comma. Since you removed the last element, you need to remove the comma after `Ice/Router.ice`, since nothing comes after it anymore.